lib/state_machine/state.hpp

```diff
@@ -5,18 +5,20 @@ namespace hyped::state_machine {
 enum class State {
   kIdle,
   kCalibrating,
```

Comment:
nitpick but stick to one case, either `calibrating`, `accelerating` etc or `calibrate`, `accelerate`